tests/plugins/test_pin_operator_digests.py

```diff
@@ -116,23 +118,48 @@ def mock_env(docker_tasker, tmpdir, orchestrator,
     return runner
 
 
-def mock_operator_csv(tmpdir, filename, pullspecs, for_ocp_44=False):
+def mock_operator_csv(tmpdir, filename, pullspecs, for_ocp_44=False, with_related_images=False):
     path = tmpdir.join(filename)
+    containers = [
+        # utils.operator adds related images as ordered dicts
+        # ("name" first, "image" second) - make sure order matches here
+        CommentedMap([('name', 'foo-{}'.format(i + 1)), ('image', image)])
+        for i, image in enumerate(pullspecs)
+    ]
     data = {
         'kind': 'ClusterServiceVersion',
         'spec': {
+            'relatedImages': [],
             # It does not really matter where in the CSV these pullspecs go
-            # as long as operator_util is known to work properly
-            'relatedImages': [
-                {'name': 'foo-{}'.format(i + 1), 'image': image}
-                for i, image in enumerate(pullspecs)
-            ]
+            # as long as utils.operator is known to work properly, just do not
+            # put them in relatedImages because worker should update those
```

Comment:
I don't understand what is meant by `as long as utils.operator is known to work properly`. Do we expect it to not work properly? 

Also, what is "worker" ?

Reply:
worker cluster?

Reply:
> I don't understand what is meant by as long as utils.operator is known to work properly. Do we expect it to not work properly?

That's just my justification for why I only put the pullspecs in `.spec.install.{...}.containers` instead of having also initContainers, RELATED_IMAGE vars etc. Basically, I'm saying that since `utils.operator` is tested relatively thoroughly, the content of the CSV file is not that important for testing the plugin.

And yeah, "worker" is worker cluster :D